### DISCUSSION TITLE
Citekey like authEtal2 but with lowercased first author (and first word in the title)

### DISCUSSION THREAD

**Discussion by thistlillo** [retorquere/zotero-better-bibtex#2352] [General]
Let's say that: 
- I have a 1999 paper titled "Long title" with two authors whose last names are First and Second. What style should I use for getting this citation key: `firstSecond99longTitle` ? Notice the first author lowercased and the first word in the title lowercased.
- I have a 1998 paper titled "Long title" with three authors whose last names are First, Second and Third. What style should I use for getting this citation key: `firstEtAl98longTitle`. As above, first words composing the author and title parts lowercased.

Is it possible to get these keys with Better BibTeX?

**Comment by retorquere** [comment]
```
authors(1,max=2).lower + authors(2) + shortyear + title.select(1,1).lower + title.select(2,1)
|
authors(1).lower  + 'EtAl' + shortyear + title.select(1,1).lower + title.select(2,1)
```
